### ISSUE TITLE
Image links should have a name attribute

### ISSUE THREAD

**Issue by wceolin** [zoonk/web#33]
**Describe the bug**
When we navigate to a post list (e.g. home page), we see a link on the thumbnail image. However, it doesn't contain an accessible name. Hence, it cannot be read by screen readers.

**To Reproduce**
1. Go to https://en.zoonk.org
2. Inspect any image from the post list
3. Accessible labels are missing

**Expected behavior**
Those links should have an `aria-label`.
